packages/frontend/editor-ui/src/components/DuplicateWorkflowDialog.vue

```diff
@@ -72,7 +78,7 @@ const save = async (): Promise<void> => {
 		return;
 	}
 
-	const parentFolderId = router.currentRoute.value.params.folderId as string | undefined;
+	const parentFolderId = props.data.parentFolderId;
```

Comment:
can we add a test for this bug? Maybe e2e here would suffice

Reply:
E2E tests added to test duplicating workflows within folders.